adapters/blip_2/blip_2.py

```diff
@@ -0,0 +1,79 @@
+import json
+from typing import List
+import PIL
+import dtlpy as dl
+import logging
+from transformers import AutoProcessor, Blip2ForConditionalGeneration
+
+logger = logging.getLogger("[BLIP]")
+CAPTIONING_PROMPT = "Caption this image."
+
+
+class HuggingAdapter:
+    def __init__(self, configuration):
+        self.model_name = configuration.get("model_name")
```

Comment:
like here, the config would be sent explicitly instead of getting it from the model entity